ui/pages/configuration/filters.js

```diff
@@ -0,0 +1,68 @@
+import { NoSsr, Paper, withStyles } from "@material-ui/core";
+import MesheryFilters from "../../components/Filters";
+import { updatepagepath } from "../../lib/store";
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux'
+import Head from 'next/head';
+import { getPath } from "../../lib/path";
+import { resolveHref } from "next/dist/next-server/lib/router/router";
+import dataFetch from "../../lib/data-fetch";
+
+const styles = {
+  paper: {
+    maxWidth: '90%',
+    margin: 'auto',
+    overflow: 'hidden',
+  }
+}
+
+class NewFilters extends React.Component {
+  componentDidMount () {
+    console.log(`path: ${getPath()}`);
+    this.props.updatepagepath({ path: getPath() });
+  }
+
+  async componentsDidMount() {
+    await new Promise(resolve => {
```

Comment:
And also this resolve part in Line 26th, you can disable the `async componentDidMount() `if the dataFetch or the API data isn't being used anywhere as of now currently. As you said, once if you decide where to put the data exactly then you can uncomment this part 👀

Reply:
Thanks, I will update this too.